src/Runner.Worker/Container/FeatureFlagManager.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using GitHub.Runner.Common;
+
+namespace GitHub.Runner.Worker.Container
```

Comment:
we might want to move this to a higher namespace since the `FeatureFlagManager` sounds like a general concept that is not scoped to the container.